<launch>
   
  <node pkg="pr2_plugs_actions" type="wiggle_plug.py" name="wiggle_plug" output="screen"/>
  <test test-name="wiggle_plug_test" pkg="pr2_plugs_actions" type="wiggle_plug_test"/>

</launch>
